subsys/bluetooth/mesh/transport.c

```diff
@@ -1001,10 +1001,8 @@ static inline int32_t ack_timeout(struct seg_rx *rx)
 	/* The acknowledgment timer shall be set to a minimum of
 	 * 150 + 50 * TTL milliseconds.
 	 */
-	to = 150 + (ttl * 50U);
-
-	/* 100 ms for every not yet received segment */
-	to += ((rx->seg_n + 1) - popcount(rx->block)) * 100U;
+	to = CONFIG_BT_MESH_SEG_ACK_BASE_TIMEOUT +
+	     (ttl * (int32_t)CONFIG_BT_MESH_SEG_ACK_PER_HOP_TIMEOUT);
```

Comment:
to will now be a constant value, and not increase as the previous code did?

Reply:
Yes, do you think this should still be in the code? It could though be configurable as well with default value 0.

Reply:
Returned that delay back, but made it configurable with default value 0.